docs/reference.md

```diff
@@ -224,48 +237,91 @@ The signature for `next` is `function(error, activation)`, where:
 
 The `whisk.getAuthKey()` function returns the authorization key under which the action is running. Usually, you do not need to invoke this function directly, because it is used implicitly by the `whisk.invoke()` and `whisk.trigger()` functions.
 
-### Runtime environment
-
-JavaScript actions are executed in a Node.js version 0.12.14 environment with the following packages available to be used by the action:
-
-- apn
-- async
-- body-parser
-- btoa
-- cheerio
-- cloudant
-- commander
-- consul
-- cookie-parser
-- cradle
-- errorhandler
-- express
-- express-session
-- gm
-- jade
-- log4js
-- merge
-- moment
-- mustache
-- nano
-- node-uuid
-- oauth2-server
-- process
-- request
-- rimraf
-- semver
-- serve-favicon
-- socket.io
-- socket.io-client
-- superagent
-- swagger-tools
-- tmp
-- watson-developer-cloud
-- when
-- ws
-- xml2js
-- xmlhttprequest
-- yauzl
+### Node.js runtime environments
+
+JavaScript actions are executed by default in a Node.js version 6.2.0 environment.  The 6.2.0 environment will also be used for an action if the kind flag is explicitly specified with a value of 'nodejs:6' when creating/updating the action.
```

Comment:
Maybe format "kind" with back ticks like `--kind` so that it is clear this is a CLI argument.